### PR TITLE
New advanced exercise on text manipulation

### PR DESCRIPTION
**Pull request by Aramist**
A (hopefully) fun exercise on searching and manipulating text from Hamlet.

### CODE REVIEW

File: advanced/README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ The tr command is used to delete or replace characters.
 echo "tata" | tr a o
 ```
 
+# Exercise: Text manipulation
+In `data/input` you will find a file called `hamlet_act_V_scene_I.txt`, which contains a scene from Shakespeare's _Hamlet_. It is a plain-text file where each character's script begins is preceeded by a line containing the character's name (Consider previewing the file with `head` or `vim` to get a feel for the format).
+## The exercise
+1) Find the number of lines spoken by "First Clown" in this scene.  _Hint: It may be useful to search the `grep` manpage for an option related to counting_
+2) Make King Claudius speak Second Clown's lines
```
Comment (kelle) [inline advanced/README.md]:
```suggestion
2) Create a new file in which King Claudius speaks the Second Clown's lines
```

File: advanced/README.md [hunk 2]
```diff
@@ -34,6 +34,12 @@ The tr command is used to delete or replace characters.
 echo "tata" | tr a o
 ```
 
+# Exercise: Text manipulation
+In `data/input` you will find a file called `hamlet_act_V_scene_I.txt`, which contains a scene from Shakespeare's _Hamlet_. It is a plain-text file where each character's script begins is preceeded by a line containing the character's name (Consider previewing the file with `head` or `vim` to get a feel for the format).
```
Comment (jsoules) [inline advanced/README.md]:
```suggestion
In `data/input` you will find a file called `hamlet_act_V_scene_I.txt`, which contains a scene from Shakespeare's _Hamlet_. It is a plain-text file where each character's dialogue is preceded by a line containing the character's name (Consider previewing the file with `head` or `vim` to get a feel for the format).
```

**Comment by jsoules** [comment]
This is a neat idea! Not sure how hard you want to make it, but it might be fun to suggest that they exchange the lines of two characters (so they have to think about how to avoid collisions--for instance, exchanging Horatio's and Hamlet's lines would require changing one of them to some third thing first). Or making sure to change the labels for the dialogue for Horatio and Hamlet, but avoid changing the text "Horatio" when it's supposed to be spoken. (Like when Hamlet talks to Horatio.)

Could also ask them to produce a count of the number of lines/speeches each character has?

**Review by jsoules — APPROVED** [review]
This looks like a good exercise--suggest we merge.

**Pull request merged**